src/react/hooks/use-chat-connection.ts

```diff
@@ -33,9 +33,75 @@ export interface UseChatConnectionResponse {
 }
 
 /**
- * A hook that provides the current connection status and error, and allows the user to listen to connection status changes.
- * @param options - The options for the hook
- * @returns The current connection status and error.
+ * React hook that provides the current connection status and error, and allows the user to listen to connection status changes.
+ *
+ * This hook automatically tracks the connection state of the underlying Ably Realtime
```

Comment:
I don't think we need to say "underlying Ably Realtime client" - its potentially just unnecessary fluff. Perhaps something more along the lines of "tracks the state of the connection between the client and Ably"